ChangeLog.d/fix-compilation-with-djgpp.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix:
```

Comment:
```suggestion
Bugfix
```
(Our [changelog assembly script](https://mbedtls.trustedfirmware.org/job/mbed-tls-pr-head/job/PR-9810-head/2/execution/node/2661/log/) is picky)